README.md

```diff
@@ -2,6 +2,9 @@
 
 This Operator is a tool for teams to integrate Bitwarden Secrets Manager into their Kubernetes workflows seamlessly.
 
+[!NOTE]  
+This is a beta release and might be missing some functionality.***
```

Comment:
I may have misunderstood the docs, I think you might need the greater-than sign in front of the lines?
```suggestion
> [!NOTE]  
> This is a beta release and might be missing some functionality.***
```